src/storage/kv/cursor.rs

```diff
@@ -323,21 +332,30 @@ impl<I: Iterator> Cursor<I> {
     ) -> Result<bool> {
         statistics.seek_for_prev += 1;
         self.mark_unread();
-        self.iter.seek_for_prev(key)
+        let perf = PerfStatisticsInstant::new();
+        let res = self.iter.seek_for_prev(key);
+        statistics.seek_for_prev_tombstone += perf.delta().0.internal_delete_skipped_count;
+        res
     }
 
     #[inline]
     pub fn next(&mut self, statistics: &mut CfStatistics) -> bool {
         statistics.next += 1;
         self.mark_unread();
-        self.iter.next().expect("Invalid Iterator")
+        let perf = PerfStatisticsInstant::new();
+        let res = self.iter.next().expect("Invalid Iterator");
+        statistics.next_tombstone += perf.delta().0.internal_delete_skipped_count;
+        res
     }
 
     #[inline]
     pub fn prev(&mut self, statistics: &mut CfStatistics) -> bool {
         statistics.prev += 1;
         self.mark_unread();
-        self.iter.prev().expect("Invalid Iterator")
+        let perf = PerfStatisticsInstant::new();
+        let res = self.iter.prev().expect("Invalid Iterator");
+        statistics.prev_tombstone += perf.delta().0.internal_delete_skipped_count;
```

Comment:
Could we calculate it after this request has been finished ?  Calculate statistics in `cursor` may be repeated with `PerfStatisticsInstant` in coprocessor.

Reply:
If it doesn't hurt performance much, I think we can just remove PerfStatisticsInstant in coprocessor

Reply:
No. Coprocessor will return content of PerfStatisticsInstant to tikv-client(TiDB). I think the frequency is too high..... You could test `tpch` which will have some large table scan.

Reply:
If we calculate it after the request is finished, it's not able to get perf stats separated by CF.

Reply:
Anyway, trying `tpch` to see the overhead